db/migrate/20170812164743_add_foreign_key_craftsman_id_to_applicant.rb

```diff
@@ -0,0 +1,20 @@
+class AddForeignKeyCraftsmanIdToApplicant < ActiveRecord::Migration
```

Comment:
We can probably rm this foreign key since we do work with it in the other branch. I'd rather not have two branches doing the same thing.